integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java

```diff
@@ -110,19 +110,19 @@ public synchronized int read(ByteBuffer buf, long size, long offset) {
     }
     final int sz = (int) size;
     int totalRead = 0;
-    int currentRead = 0;
+    int currentRead;
     try {
       mInStream.seek(offset);
-      while (currentRead >= 0 && totalRead < sz) {
+      do {
```

Comment:
just wonder why we change some while loop to do while?

Reply:
int currentRead = 0 
- while(currentRead >=0) currentRead = instream.read()
- do currentRead = instream.read while (currentRead > 0)
The first approach is correct in normal cases, just afraid that some instream.read() returns 0 forever and causes we stuck in the while loop forever (although normally it wont)
The second approach can make sure we do not continue read while inStream.read returns 0 or -1